snapshots/artificial_intelligence/2025-11-07/energy_ai_iea.xlsx.dvc

```diff
@@ -0,0 +1,33 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Energy and AI
+    date_published: "2025-06-16"
+    description_snapshot: |
+      This dataset contains world and regional data on energy consumption and infrastructure related to artificial intelligence, including:
+      - Installed capacity
+      - Power usage effectiveness (PUE)
+      - Load factor
+      - Electricity consumption
+
+      The data provides insights into the energy requirements and efficiency of AI infrastructure globally.
+
+    # Citation
+    producer: International Energy Agency
+    citation_full: "International Energy Agency (IEA), 'Energy and AI'. Published online at iea.org. Retrieved from: 'https://www.iea.org/data-and-statistics/data-product/energy-and-ai' [online resource]"
+
+    # Files
+    url_main: https://www.iea.org/data-and-statistics/data-product/energy-and-ai
+    url_download: https://www.iea.org/product/download/020879-000396-020495
```

Comment:
This link doesn't work, the snapshot loads a local file. So, just to avoid confusion, I'd remove the url_download.